src/raftstore/store/peer.rs

```diff
@@ -69,6 +69,15 @@ impl ReadIndexRequest {
     }
 }
 
+impl Drop for ReadIndexRequest {
+    fn drop(&mut self) {
+        let dur = (monotonic_raw_now() - self.renew_lease_time)
```

Comment:
What about adding a `HistgromTimer` in ReadIndexRequest?

Reply:
A `Timer` needs to get system time 2 time. Use `renew_lease_time` as the start time can save a call.